src/stan-users-guide/odes.Rmd

```diff
@@ -4,30 +4,35 @@ Stan provides four different methods for solving systems of ordinary
 differential equations (ODEs). All of these methods will adaptively
 refine their solutions in order to satisfy the given tolerances.
 
-The upside to this is that as long as reasonable tolerances are provided
-and an ODE solver well-suited to the problem is chosen, the choice of timestep
-can be abstracted away. The downside is that it is not always clear from the
-outset what reasonable tolerances are, or which ODE solver is best suited
-to each problem. Additionally, in the course of working with an ODE model in
-Stan the correct answer may change (for instance, the ODE may change).
+Each solver has a simple interface with default tolerances and then an advanced
+one that allows the tolerances to be modified. If components of the solution
+need to be solved with high precision or if the solution goes very near zero,
```

Comment:
We may want to adjust wording to avoid confusion: if a low(like 1.e-4) solution is expected one can also choose `_tol` solvers.

Reply:
I don't want to encourage the coarse tolerance -> high performance thing and I'm also sketched out about low precision gradients for HMC.

Reply:
(So I don't want to encourage lower than default tolerances)

Reply:
we doubt we have a lot of evidence to back the default tolerances. In many occasions they're simply not reasonable. In lotka-volterra model the tol is set to 1.e-3: how would one justify measuring the number of hares up to 1.e-6 precision?

Reply:
Mm, fair, I changed it to:

```
If components of the solution need to be solved with higher precision, the
solution can be solved with lower than default tolerances, or if the solution
goes very near zero, then the advanced interface may be useful.
```